dashboard/lib/routes/subscribe.js

```diff
@@ -149,18 +149,21 @@ function searchSubscription(user, regions, callback) {
 
 /**
  * notify to region list for a change in region
- * @param {String} region
+ * @param {Object} region
  * @param {function} notifyCallback
  */
 function notify(region, notifyCallback) {
 
-    logger.info({op: 'subscribe#notify'}, 'notify to region: ' + region);
+    var regionName = region.node,
+        regionStatus = region.status;
 
+    logger.info({op: 'subscribe#notify'}, 'notify to region: ' + regionName);
 
     var payloadString = 'name_from= fi-health sanity&';
         payloadString += 'email_from=' + config.mailman.emailFrom + '&';
-        payloadString += 'subject=Status changed for region ' + region + '&';
-        payloadString += 'body=Status changed for region ' + region;
+        payloadString += 'subject=Status changed for region ' + regionName + '&';
+        payloadString += 'body=Status changed to ' + regionStatus + ' for region ' + regionName +
+                         ' (visit ' + config.fiHealthUrl + ' for details)';
```

Comment:
We are already sending the URL in the body of the message. It is defined in the Mailman template we use.